deps/GraphBLAS/Source/GxB_Matrix_tuple_iter.c

```diff
@@ -149,6 +156,10 @@ GrB_Info GxB_MatrixTupleIter_next
 
 	if(col)
 		*col = A->i[nnz_idx] ;
+	if(val) {
+		GrB_Index offset = nnz_idx * iter->size;
+		memcpy(val, (char*)A->x + offset, iter->size);
```

Comment:
Is this valid for all matrix format supported?
what happened if the matrix isn't flushed?

Reply:
An iterator must be operating on a synced matrix, otherwise behavior isn't defined, caller's responsibility.